tool/plugin/lib/plugin.dart

```diff
@@ -432,6 +432,13 @@ class ArtifactManager {
           log('download failed');
           break;
         }
+        var archiveFile = new File(path);
+        if (!archiveFile.existsSync() || archiveFile.lengthSync() < 200) {
+          log('archive file not found: $base/${artifact.file}');
```

Comment:
nit: why do files less than length 200 need to be treated as missing?

Reply:
If the file is missing then the downloader returns an XML file, length 127, that contains an error message. I considered analyzing the XML to make sure it was reporting a missing-file error, but didn't bother. The files are all quite large.